### PR TITLE
domains are appended to host file

### PR DESCRIPTION
**Pull request by jan-kowalczyk-wttech**

### CODE REVIEW

File: pkg/project/app_classic/local.env [hunk 1]
```diff
@@ -10,4 +10,8 @@ AEM_PUBLISH_PASSWORD=admin
 AEM_PUBLISH_HTTP_URL=http://127.0.0.1:4503
 AEM_PUBLISH_DEBUG_ADDR=0.0.0.0:14503
 
-AEM_DISPATCHER_HTTP_URL=http://127.0.0.1
+AEM_DISPATCHER_IP=127.0.0.1
+AEM_DISPATCHER_HTTP_URL=http://${AEM_DISPATCHER_IP}
```
Comment (krystian-panek-wttech) [inline pkg/project/app_classic/local.env:14]: how that may work? there is no interpolation here like in Ansible

File: pkg/project/app_classic/local.env [hunk 2]
```diff
@@ -10,4 +10,8 @@ AEM_PUBLISH_PASSWORD=admin
...
+
+AEM_DOMAINS=host.docker.internal author.aem.local
```
Comment (krystian-panek-wttech) [inline pkg/project/app_classic/local.env:16]: why `host.docker.internal` here? it's rather managed itself by Docker

File: pkg/project/app_cloud/local.env [hunk 3]
```diff
@@ -10,4 +10,8 @@ AEM_PUBLISH_PASSWORD=admin
 AEM_PUBLISH_HTTP_URL=http://127.0.0.1:4503
 AEM_PUBLISH_DEBUG_ADDR=0.0.0.0:14503
 
-AEM_DISPATCHER_HTTP_URL=http://127.0.0.1
+AEM_DISPATCHER_IP=127.0.0.1
+AEM_DISPATCHER_HTTP_URL=http://${AEM_DISPATCHER_IP}
+
+AEM_DOMAINS=host.docker.internal author.aem.local
```
Comment (krystian-panek-wttech) [inline pkg/project/app_cloud/local.env:16]: rather `publish` is needed here which is an expected value by dispatcher config

File: pkg/project/app_classic/local.env (hunk 2 above)
Comment (jan-kowalczyk-wttech) [reply pkg/project/app_classic/local.env:16]: You have right, just my mistake

Comment (jan-kowalczyk-wttech) [inline pkg/project/app_classic/local.env:16]:
```suggestion
AEM_DOMAINS=publish.aem.local author.aem.local
```

File: pkg/project/app_cloud/local.env (hunk 3 above)
Comment (jan-kowalczyk-wttech) [inline pkg/project/app_cloud/local.env:16]:
```suggestion
AEM_DOMAINS=publish.aem.local author.aem.local
```

**Pull request merged**